src/components/PeoplePage.tsx

```diff
@@ -1,11 +1,39 @@
+import { useCallback, useEffect, useState } from 'react';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
 
 export const PeoplePage = () => {
+  const [listPeople, setListPeople] = useState<Person[]>([]);
+  const [isNotFound, setIsNotFound] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then((res: Person[]) => {
+        if (!res.length) {
+          setIsError(true);
```

Comment:
the absence of items isn't an error, add message if length of array === 0
when I got an error see that message
<img width="1053" alt="Screenshot 2023-03-21 at 17 55 04" src="https://user-images.githubusercontent.com/53228259/226666036-a41c758c-5573-4400-a06f-747b1f53e1a0.png">